src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(value) {
+      calculator.result += value;
+    },
```

Comment:
Methods `add`, `subtract`, `multiply` and `divide` shouldn't know anything about the `result` property, the should accept 2 arguments and return some result depending on their own functionality 🙃